### PR TITLE
Increase priority of release channel builds for devicelab and framework builders.

### PR DESCRIPTION
**Pull request by christopherfujino**
Increase priority of release channel builds for devicelab and framework builders.

cc @pcsosinski

### CODE REVIEW

File: config/framework_config.star [hunk 1]
```diff
@@ -157,6 +157,9 @@ def framework_prod_config(branch, version, testing_ref, release_ref):
             max_concurrent_invocations = 3,
         )
 
+    # Defines build priority, release builds should be prioritized.
+    priority = 30 if branch == "master" else 25
```
Comment (godofredoc) [inline config/framework_config.star:161]: Can we make it 29 to ensure they are not prioritized over retries and or subbuilds?

File: config/devicelab_config.star [hunk 2]
```diff
@@ -144,12 +144,14 @@ def devicelab_prod_config(branch, version, ref):
...
             max_concurrent_invocations = 1,
         )
+        priority = 30
     else:
         triggering_policy = scheduler.greedy_batching(
             max_batch_size = 1,
             max_concurrent_invocations = 3,
         )
         devicelab_triggering_policy = triggering_policy
+        priority = 25
```
Comment (godofredoc) [inline config/devicelab_config.star:154]: Can we make it 29 to ensure they are not prioritized over retries and or subbuilds?

File: config/framework_config.star (hunk 1 above)
Comment (christopherfujino) [reply config/framework_config.star]: done

File: config/devicelab_config.star (hunk 2 above)
Comment (christopherfujino) [reply config/devicelab_config.star]: done

**Review by godofredoc — APPROVED** [review]

**Pull request merged**